src/views/landing.njk

```diff
@@ -37,4 +36,13 @@
       <p class="govuk-body-s">Read the <a href="https://www.gov.uk/government/publications/company-strike-off-dissolution-and-restoration/strike-off-dissolution-and-restoration" target="_blank" class="govuk-link">guidance for striking off and dissolving a company (opens in a new tab)</a>.</p>
     </div>
   </div>
+  {% if piwik %}
+    <script nonce={{ nonce | dump | safe }}>
+      $(document).ready(function() {
```

Comment:
We should be able to write this (and the other embedded JS) using arrow functions instead.

```
$(document).ready(() => {
    $('#start-now').on('click', () => {
         _paq.push(["trackGoal", {{ piwik.landingPageStartGoalId }}]);
    });
})
```